data/synapse-data-db2/pom.xml

```diff
@@ -23,10 +23,27 @@
     <modelVersion>4.0.0</modelVersion>
     <artifactId>synapse-data-db2</artifactId>
 
+    <properties>
+        <synapse.version>0.3.30-SNAPSHOT</synapse.version>
+        <hibernate.core.version>5.6.15.Final</hibernate.core.version>
+        <hibernate.entitymanager.version>5.6.15.Final</hibernate.entitymanager.version>
+    </properties>
+
     <dependencies>
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-data-jpa</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.apache.tomcat</groupId>
+                    <artifactId>tomcat-jdbc</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>io.americanexpress.synapse</groupId>
+            <artifactId>synapse-utilities-common</artifactId>
+            <version>${synapse.version}</version>
```

Comment:
remove version, this should already be specify in the parent pom

Reply:
done